lib/improver/nbhood/circular_kernel.py

```diff
@@ -462,9 +465,13 @@ def make_percentile_cube(self, cube):
                 Each slice along this coordinate is identical.
         """
         pctcubelist = iris.cube.CubeList()
+        pct_coord_name = "percentiles_over_neighbourhood"
         for pct in self.percentiles:
             pctcube = cube.copy()
             pctcube.add_aux_coord(iris.coords.DimCoord(
-                pct, long_name="percentiles_over_neighbourhood", units='%'))
+                pct, long_name=pct_coord_name, units='%'))
             pctcubelist.append(pctcube)
-        return pctcubelist.merge_cube()
+        result = pctcubelist.merge_cube()
+        if result.coord_dims(pct_coord_name) == ():
```

Comment:
Changes look good and seem to address the problem. Unit/CLI tests etc pass...

Minor comment - just because I overheard you talking to Anna about this: so we don't forget when we come back to look at this code in a few years time, could you perhaps put a comment about here explaining why this extra work is required? e.g: "A single percentile would be a scalar coord (coord_dims == ()) - promoting this to a dimension coord" or similar?

Reply:
Done